.github/workflows/ci.yml

```diff
@@ -5,16 +5,20 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    services:
+      trino:
+        image: trinodb/trino:355
+        ports:
+          - 8080:8080
```

Comment:
Does this definition wait until Trino server is ready?

Reply:
Explicit check would help avoid flakiness if possible. We already have a lot of it. 😞

Reply:
Added the `trino_start` fixture to solve this.